src/server/scripts/Commands/cs_go.cpp

```diff
@@ -32,6 +32,7 @@ class go_commandscript : public CommandScript
             { "graveyard",      SEC_MODERATOR,      false, &HandleGoGraveyardCommand,         "" },
             { "grid",           SEC_MODERATOR,      false, &HandleGoGridCommand,              "" },
             { "object",         SEC_MODERATOR,      false, &HandleGoObjectCommand,            "" },
```

Comment:
Just wondering if we should've added a "deprecated" comment here
```suggestion
            { "object",         SEC_MODERATOR,      false, &HandleGoObjectCommand,            "" }, // DEPRECATED: use gobject instead
```